server/container/guice/guice-common/src/main/java/org/apache/james/modules/OptionalModuleLoader.java

```diff
@@ -0,0 +1,57 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.modules;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.apache.commons.io.IOUtils;
+import org.apache.james.modules.server.WebAdminServerModule;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.inject.AbstractModule;
+
+public class OptionalModuleLoader extends AbstractModule {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OptionalModuleLoader.class);
+
+    @Override
+    protected void configure() {
+        try {
+            if (isWebAdminEnabled()) {
+                install(new WebAdminServerModule());
+            } else {
+                LOGGER.info("WebAdmin server disabled by configuration");
+            }
+        } catch (Exception e) {
```

Comment:
misplace catch, what if something else than configuration reading fail ?
And e.printStackTrace() is not a good idea